presto-main/src/test/java/io/prestosql/sql/planner/TestPlanMatchingFramework.java

```diff
@@ -257,4 +260,11 @@ public void testProjectLimitsScope()
                         project(ImmutableMap.of("EXPRESSION", expression("CAST(1 AS bigint) + ORDERKEY")),
                                 tableScan("lineitem", ImmutableMap.of("ORDERKEY", "orderkey")))));
     }
+
+    private Session disableCbo()
+    {
+        return Session.builder(getQueryRunner().getDefaultSession())
+                .setSystemProperty(JOIN_DISTRIBUTION_TYPE, JoinNode.DistributionType.PARTITIONED.name())
```

Comment:
If this method is supposed to disable CBO, it should also set `JOIN_REORDERING_STRATEGY=` to eg NONE